client/geo.py

```diff
@@ -0,0 +1,44 @@
+import re
+import time
+
+import geopy.geocoders
+from django.conf import settings
+
+
+class Locators:
+    NOMINATIM = 'NOMINATIM'
+    GOOGLE = 'GOOGLE'
+    ARCGIS = 3
```

Comment:
Why are the two others options strings and this one is a number?
Also looks like this can become an Enum